troposphere/static/js/components/images/common/SecondaryImageNavigation.react.js

```diff
@@ -39,7 +39,7 @@ export default React.createClass({
 
 
             if (!userImages || !favoritedImages) {
-                return <div className="loading"></div>
+                return <div style={{ height: "63px" }} className="secondary-nav"/>
```

Comment:
This may look strange but I am rendering the `secondary-nav` without links as a place holder until we have data because the response time is slow and otherwise causes the content to jump around.